docs/general/introduction.md

```diff
@@ -1,81 +1,101 @@
+The [`README.md`](https://github.com/mxstbr/react-boilerplate#features) gives you adequate information on how to clone boilerplate files, install dependencies and launch the example app.
 
-## Foreword
-This boilerplate will give you the best DX (Developer eXperience). But it won't make it easier on beginners. It's supposed to make it easier on browsers. The boilerplate can be a little daunting but it shouldn't have to be. Before you get your hands dirty with the source code, we'd like you to go through a checklist, that will help you determine whether or not you're eligible to use this boilerplate. It's not because we're _holier-than-thou_, but we genuinely want to save you the frustration.
+Once you have done that, this document is intended to give you a taste of how `react-boilerplate` works. It still assumes basic knowledge of React, Redux and `react-router`. **If you're completely new to React, please refer to https://github.com/petehunt/react-howto instead!**
+
+This is a production-ready boilerplate, and as such optimized for browsers, not for beginners. It includes tools to help you manage performance, asynchrony, styling, everything you need to build a _real_ application. Before you get your hands dirty with the source code, we'd like you to go through a checklist, that will help you determine whether or not you're eligible to use this boilerplate. It's not because we're _holier-than-thou_, but we genuinely want to save you the frustration.
+
+> Opening an issue is the fastest way to draw the attention of the team, but please make it a point to read the [docs](https://github.com/mxstbr/react-boilerplate/tree/master/docs) and [contribution instructions](https://github.com/mxstbr/react-boilerplate/blob/master/.github/CONTRIBUTING.md) before you do. The issues section is specifically used for pointing out defects and suggesting enhancements, if you have a question about one of the tools please refer to StackOverflow instead.
 
 ## Tech Stack
-Here's a curated list of packages (in no particular order) that you should have in-depth knowledge of, before starting your awesome project. However, the best way to have a complete list of dependencies is to see [package.json](https://github.com/mxstbr/react-boilerplate/blob/master/package.json).
+
+Here's a curated list of packages (in no particular order) that you should have knowledge of, before starting your awesome project. However, the best way to have a complete list of dependencies is to see [package.json](https://github.com/mxstbr/react-boilerplate/blob/master/package.json).
```

Comment:
> (in no particular order)

I know I wrote this but it seems unnecessary now that I read it. Also, logically if you don't know React, you won't understand react router and styled components. So the order does matter here. But it's not important enough to highlight.